src/components/screens/ProfileScreen.js

```diff
@@ -1,24 +1,169 @@
-import React from 'react';
-import {SafeAreaView, Text} from 'react-native';
+import React, {useState} from 'react';
+import {
+  SafeAreaView,
+  Text,
+  StyleSheet,
+  View,
+  TouchableOpacity,
+} from 'react-native';
+import Slider from '@react-native-community/slider';
+import CheckBox from '@react-native-community/checkbox';
 import {connect} from 'react-redux';
-import ApproachButton from '../ApproachButton';
 import {dispatchClearUserDetailsAction} from '../../actions/userActions';
 import * as authService from '../../services/authService';
+import ApproachText from '../ApproachText';
+import {useTheme} from '@react-navigation/native';
 
-export const ProfileScreenName = 'profile';
-export const ProfileIcon = require('../../../assets/icons/profile.png');
-export const ProfileIconColored = require('../../../assets/icons/profile-colored.png');
+function ProfileScreen({clearUserDetailsFromLocalStorage, user}) {
+  const {
+    name: name,
+    age: age,
+    photos: photos,
+    location: location,
+    position: position,
+    about: about,
+    labels: labels,
+    filters: filters,
+  } = user;
```

Comment:
```suggestion
  const {
    name,
    age,
    photos,
    location,
    position,
    about,
    labels,
    filters,
  } = user;
```
The `: field` in this case is redundant since you're renaming it to its original name. IMO, I'd just use `user.field` wherever it's used, it makes it more clear as to what it is since it's being used away from this deconstruction scope.